### DISCUSSION TITLE
Kafka interval suppression alarm

### DISCUSSION THREAD

**Discussion by llnbe** [prometheus/prometheus#11221] [Q&A]
Whether to suppress alarms within time intervals, for example, do not send alarm emails between 23:00 and 9:00

expr: changes(kafka_topic_partition_current_offset{topic!="__consumer_offsets"}[10m]) == 0

23:00-9:00 No alarm Don't know how to achieve

**Comment by LeviHarrison** [comment]
You can set `mute_time_intervals` in your alertmanager config. See https://prometheus.io/docs/alerting/latest/configuration/#route.
